### PR TITLE
Update bower.json

### PR DESCRIPTION
**Pull request by braxtondiggs**
JS path change

### CODE REVIEW

**Comment by shawila** [comment]
:+1:

**Comment by shawila** [comment]
Actually, is there a way to specify whether you want 'dist/scripts/xdLocalStorage.min.js` or `dist/scripts/ng-xdLocalStorage.min.js`?

**Comment by braxtondiggs** [comment]
Put in a bower override:
https://roots.io/using-bootstrap-with-bower-how-to-override-bower-packages/

File: bower.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "Ofir Dagan <ofirdagan@gmail.com>"
   ],
   "dependencies": {},
-  "main": "scripts/xdLocalStorage.js",
+  "main": "dist/scripts/xdLocalStorage.min.js",
```
Comment (ofirdagan) [inline bower.json]: Hi @braxtondiggs , I'm sorry for the big delay from some reason the github mail reached my spam folder.. Now it's fixed.

Regarding your change, according to bower (http://bower.io/docs/creating-packages/) you should not reference minified files in the main section. Why do u need it?

**Comment by braxtondiggs** [comment]
I don't think you have to use the minified JS but your path is still incorrect. The script path is no longer available when you pull down your code from GitHub.

**Comment by ofirdagan** [comment]
gotcha :) I fixed it now. Thanks!!

**Comment by braxtondiggs** [comment]
Thanks!

**Pull request closed without merging**